### PR TITLE
Patch entity spawns

### PR DESCRIPTION
**Pull request by svdgoor**

### CODE REVIEW

**Review by StrangeOne101 — APPROVED** [review]
Approved but I want a second opinion on whether the try catch should go here before we merge :+1:

**Comment by svdgoor** [comment]
Reason for this is that someone had their server literally crash because of the get real entity

**Comment by StrangeOne101** [comment]
Yeah I get that it does bring the server to its knees. It's probably fine so I'll just merge for now

**Pull request merged**
